### PR TITLE
✨ Add badges

### LINKED ISSUE #81: Create badge
Image

Without icons

### PR DESCRIPTION
**Pull request by iurynogueira**
fix #81 

image

### CODE REVIEW

File: projects/ion/src/lib/badge/badge.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<span [class]="'badge-' + type">{{ valueInBadge }}</span>
```
Comment (AlyssonMascarenhas) [inline projects/ion/src/lib/badge/badge.component.html:1]: Missing of 'type' in ts.

File: projects/ion/src/lib/badge/badge.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+@Component({
+  selector: 'ion-badge',
+  templateUrl: './badge.component.html',
+  styleUrls: ['./badge.component.scss'],
+})
+export class BadgeComponent {
+  @Input() label?: string;
+  @Input() value?: number;
+
```
Comment (lucasrasec) [inline projects/ion/src/lib/badge/badge.component.ts:21]: I think the "type" variable is missing here

**Review by AlyssonMascarenhas — APPROVED** [review]
Good job!

**Review by lucasrasec — APPROVED** [review]

File: projects/ion/src/lib/badge/badge.component.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+})
+export class BadgeComponent {
+  @Input() label?: string;
+  @Input() value?: number;
+  @Input() type!: BadgeType;
+
+  public valueInBadge: string;
+
+  private exists(value: number): boolean {
+    return value !== undefined && value !== null && !isNaN(value);
```
Comment (mizek1) [inline projects/ion/src/lib/badge/badge.component.ts:26]:
```suggestion
    return value !== null && !isNaN(value);
```
`isNaN` already validates undefined values. (more about it [here](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/isNaN))

**Pull request merged**